lib/bolt/executor.rb

```diff
@@ -8,6 +8,7 @@
 require 'set'
 require 'bolt/analytics'
 require 'bolt/result'
+require 'bolt/task'
```

Comment:
The executor doesn't use this, I'm tempted to leave it out of the requires.

Reply:
And include 'bolt/task' directly in files that use it?